apps/expo/src/components/OutlinedButtons.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Text, TouchableOpacity, View } from "react-native";
+
+interface OutlinedButtonProps {
+  onPress: () => void;
+  children: React.ReactNode;
+  fill: boolean;
+  // icon to be added
+}
+
+const OutlinedButton: React.FC<OutlinedButtonProps> = ({
+  onPress,
+  children,
+  fill,
+}) => {
+  return (
+    <TouchableOpacity onPress={onPress}>
+      <View className="h-12 flex-auto items-start gap-2">
+        <View
+          className={`border-p-40 rounded-full border bg-white px-6 py-2 ${
```

Comment:
is there a better way to center the text than by using padding?

Reply:
I tried a lot of methods like `items-center`, `content-center`, messing with gap and width, structuring the views differently, but couldn't find any other way without the button losing its structure TT. Do you know any way to solve this issue?

Reply:
ok thats fine, as long as you tried other options